src/WpfMath.Tests/WpfMath.Tests.fsproj

```diff
@@ -1,31 +1,29 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
+﻿<Project Sdk="Microsoft.NET.Sdk.WindowsDesktop">
     <PropertyGroup>
-        <TargetFramework>net461</TargetFramework>
+        <TargetFrameworks>netcoreapp3.0;net461</TargetFrameworks>
         <IsPackable>false</IsPackable>
+        <LangVersion>4.7</LangVersion>
+        <UseWpf>true</UseWpf>
         <DebugType>Full</DebugType>
         <Deterministic>false</Deterministic>
     </PropertyGroup>
     <ItemGroup>
-        <PackageReference Include="ApprovalTests" Version="3.0.19" />
-        <PackageReference Include="Foq" Version="1.7.3" />
-        <PackageReference Include="FSharp.Core.Fluent-4.0" Version="1.0.0.6" />
```

Comment:
Why have you removed `FSharp.Core.Fluent`? It is a good package, I like it :(
Does it have some compatibility troubles? It shouldn't.

Reply:
I first trying to remove `FSharp.Core.Fluent-4.0` to resolve the problem of `System.ValueTuple` version conflicts between 4.4.0 and 4.5.0, however after that I found Foq also referenced the 4.4.0 version....

But sure I can definitely add `FSharp.Core.Fluent-4.0` back again.

Reply:
Update: added back via https://github.com/ForNeVeR/wpf-math/pull/219/commits/a08a4e8294c25cff12b4607493398ea80aea2089